webpack/sessionStorage.js

```diff
@@ -0,0 +1,21 @@
+export const loadState = () => {
+  // returning undefined will cause redux to use the default values
+  try {
+    const serializedState = sessionStorage.getItem("state");
+    if (serializedState === null) {
+      return undefined;
+    }
+    return JSON.parse(serializedState);
+  } catch (err) {
+    return undefined;
+  }
+};
+
+export const saveState = state => {
+  try {
+    const serializedState = JSON.stringify(state);
+    sessionStorage.setItem("state", serializedState);
+  } catch (err) {
+    // write error
```

Comment:
Is the comment to "write error" here in place of something in the future?

Reply:
Not really, it's just in lieu of a `pass` statement, since Javascript doesn't have one.  There's nothing to be done if it fails to write (a UI notification doesn't seem appropriate).  See https://eslint.org/docs/rules/no-empty :)